jupyter_book/default_config.yml

```diff
@@ -13,6 +13,7 @@ logo                        : ""  # A path to the book logo
 exclude_patterns            : [_build, Thumbs.db, .DS_Store, "**.ipynb_checkpoints"]
 # Auto-exclude files not in the toc
 only_build_toc_files        : false
+use_multitoc_numbering      : true
```

Comment:
add a comment here with a brief description of the option

Reply:
Thanks @chrisjsewell . Done in https://github.com/executablebooks/jupyter-book/pull/1248/commits/19332e36b6cd6d8c4b526755b2c3693e548e850d